### PR TITLE
Twitter sample

### PR DESCRIPTION
**Pull request by corycook**
This adds the Twitter sample code to display input and output type inheritance.

### CODE REVIEW

**Comment by zgrossbart** [comment]
It would be really great of we had some sample queries in the README.

There should be an error if you haven't created the `.env` file.  It would also be great if that file didn't have a hidden name.

We should also add a comment explaining what each file does.  This is a sample and we're trying to make it clear why we have multiple files.

File: examples/twitter/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- TimelineInput ([inputs/TimelineInput.ts](inputs/TimelineInput.ts)): input fields for user timelines.
+
+This solution also has some helper functions that are kept in a _lib_ folder:
+
+- getToken ([lib/getToken.ts](lib/getToken.ts)): fetch an access token for the Twitter API
+- makeTweet ([lib/makeTweet.ts](lib/makeTweet.ts)): this is a factory function to generate the correct output Tweet type (Retweet or Tweet) based on the data returned from the API.
+
+## @context argument vs @context field
+
+This example binds the context as both an argument (in Twitter.ts) and as a field (in User.ts).
```
Comment (zgrossbart) [inline examples/twitter/README.md]: I'm a little confused by this context.  Is there a place where we say what a context is and why we need it?

**Comment by zgrossbart** [comment]
This example looks great.  I haven't been able to test it because I'm still waiting for the Twitter API key, but I looked at the code and made a few line edits on the README.

File: examples/twitter/README.md (hunk 1 above)
Comment (corycook) [reply examples/twitter/README.md]: Yeah, I'll add some documentation for context.

**Pull request merged**